### PR TITLE
creating to do items

### PR DESCRIPTION
**Pull request by erstrong**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,4 +3,7 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
+
+has_many :items
```
Comment (bridgetlandis) [inline app/models/user.rb]: tab this over a bit - should be in line with devise

File: app/views/items/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      <h4>There are <%= pluralize(@item.errors.count, "error") %>.</h4>
+      <ul>
+        <% @item.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="form-group #{error_tag(@item.errors[:body]}" >
```
Comment (bridgetlandis) [inline app/views/items/_form.html.erb]: seems like this should be item.errors instead of @item. - is that working?

Comment (bridgetlandis) [inline app/views/items/_form.html.erb]: ahh i see - so in your show page its @item and here you're using item - to be consistent lets use item here.

File: app/models/user.rb (hunk 1 above)
Comment (erstrong) [inline app/models/user.rb]: fixed!

File: app/views/items/_form.html.erb (hunk 2 above)
Comment (erstrong) [inline app/views/items/_form.html.erb]: Fixed, but now I'm noticing that <% if item.errors.any? %> is always returning false. The page will successfully throw an error but it doesn't show the error messages in the if logic. Also doesn't show a test block of text I inserted.

**Pull request closed without merging**